app/javascript/components/admin/site_settings/administration/LinksForm.jsx

```diff
@@ -17,17 +15,16 @@ export default function LinksForm({ id, value, mutation: useUpdateSiteSettingsAP
 
   return (
     <Form id={id} methods={methods} onSubmit={updateSiteSettingsAPI.mutate}>
-      <InputGroup>
-        <FormControlGeneric
-          field={fields.value}
-          aria-describedby={`${id}-submit-btn`}
-          type="text"
-        />
-        <Button id={`${id}-submit-btn`} variant="brand" type="submit" disabled={updateSiteSettingsAPI.isLoading}>
-          {updateSiteSettingsAPI.isLoading && <Spinner className="me-2" />}
-          { t('admin.site_settings.administration.change_url') }
-        </Button>
-      </InputGroup>
+      <FormControl
+        field={fields.value}
+        aria-describedby={`${id}-submit-btn`}
+        type="text"
+        noLabel
+      />
+      <Button id={`${id}-submit-btn`} variant="brand" type="submit" disabled={updateSiteSettingsAPI.isLoading} className="mb-2 float-end">
```

Comment:
Not important, but could you move `className` to be right after `id`. It's just aesthetically nicer to read/good practice